src/components/Pages/ManageWiki/Tabs/ManageFeatures.vue

```diff
@@ -1,13 +1,22 @@
 <template>
   <v-content>
-      Feature Settings
+    <FederatedProperties :wikiId="this.wikiId" />
   </v-content>
 </template>
 
 <script>
+import FederatedProperties from '../Features/FederatedProperties.vue'
+
 export default {
   name: 'ManageFeatures',
-  components: {
+  components: { FederatedProperties },
+  data () {
+    return {
+      wikiId: 0
```

Comment:
I feel like `null` is more appropriate as an initial value. `0` might be an actual wiki id.

```suggestion
      wikiId: null
```

Reply:
done!